test/enigma_test.rb

```diff
@@ -12,6 +12,11 @@ def test_it_exists
     assert_instance_of Enigma, @enigma
   end
 
+  def test_it_can_encrypt
+    expected = {encryption: "keder ohulw", key: "02715", date: "040895"}
+    assert_equal expected , @enigma.encrypt("hello world", "02715", "040895")
+  end
+
   def test_it_can_encrypt_without_date
     Timecop.freeze(Time.local(2020,4,19))
```

Comment:
found on stack overflow to freeze time? Allowed me to move around without having a mock.